_components/customisation.md

```diff
@@ -0,0 +1,199 @@
+</code></pre>
+
+
+<figure>
+	<img alt="" src="/assets/images/component-customisation/whitelabel-customisation.png" />
+	<figcaption>
+        An example project which builds on the whitelabel brand. The font has been customised with `o-typography` Sass (oooh! Comic Sans). It affects all components which use `o-typography`, including the other components on the page, `o-table` and `o-header-services`. `o-header-services` has also been customised using Sass to have a unique highlight colour (oooh! hotpink!)
+	</figcaption>
+</figure>
+
+This is great to create a distinct brand, where a customisation should affect all instances of a component across a project.
+
+If your project is based on the master or internal brand, however, it is more likely that you want to create new variants of a component to use alongside existing ones. For example to output an additional banner style to support a new marketing campaign.
+
+## Themes/Variants
+
+If your project has access to a component's Sass (see the [manual build tutorial](/docs/tutorials/manual-build/#selecting-a-brand)) your project may use Sass mixins to customise components by outputting a new variant of that component. A component variant is an instance of a component with modified appearance or functionality.
+
+For instance, at the time of writing, the `o-banner` component has an [`oBannerAddTheme` mixin](https://registry.origami.ft.com/components/o-banner@3.4.2/sassdoc?brand=master#mixin-obanneraddtheme) which provides options for creating a banner theme with customised colour and background image:
+
+<pre><code class="o-syntax-highlight--scss">@include oBannerAddTheme('my-pikachu-theme', (
+    background-image: url('https://example.org/pikachu.png'),
+    background-color: oColorsByName('lemon'),
+    text-color: oColorsByName('slate'),
+    button-background-color: oColorsByName('crimson'),
+));</code></pre>
+
+<aside>If your project is using the <a href="/docs/tutorials/build-service/">Origami Build Service</a>, and does not have access to the component's Sass API, it is not currently possible to create custom component variants. It may be possible in the future using <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties">Custom CSS Properties (CSS Variables)</a>, when projects no longer support older browsers such as Internet Explorer.</aside>
+
+<figure>
+	<img alt="" src="/assets/images/component-customisation/pikachu-banner.png" />
+	<figcaption>
+        A custom banner theme featuring a Pikachu background (this is a fun example but a bad one because Pikachu's ear obscures some text).
+	</figcaption>
+</figure>
+
+## Avoid CSS Overrides
+
+Another way to customise components is by applying additional CSS styles within your project. This is not recommended and should be avoided. Applying styles on top of a components style is unreliable and should be avoided as component CSS may change in future minor releases in a way that conflicts with your overrides.
```

Comment:
Maybe we should add emphasis on "This is not recommended and should be avoided."

We could also add an aside which repeats that this is not recommended, to really drive the point, since we really recommended not using this approach

Reply:
It couldn't hurt! I've emphasised "not recommended" and added an aside to offer support in removing overrides.

![Screenshot 2021-03-19 at 16 14 29](https://user-images.githubusercontent.com/10405691/111810802-4eb41f80-88ce-11eb-9641-d5710303c2bc.png)

Reply:
The aside now includes more English
![Screenshot 2021-03-19 at 16 16 04](https://user-images.githubusercontent.com/10405691/111810934-71463880-88ce-11eb-9c18-10601215c555.png)